branch/src/main/java/com/rudderstack/android/integration/branch/BranchIntegrationFactory.java

```diff
@@ -180,34 +196,46 @@ public void dump(RudderMessage element) {
                                 BranchEvent ua_be = new BranchEvent(BRANCH_STANDARD_EVENT.UNLOCK_ACHIEVEMENT);
                                 this.logEventToBranch(ua_be, property);
                                 break;
+                            case "Invite":
```

Comment:
Why can't we use "case BRANCH_STANDARD_EVENT.INVITE"?

Reply:
We are trying to map from RudderStack standard event to Branch event and we don't have `Invite` as a standard event in RudderStack, therefore I didn't utilise it.